app/Locale/Slovak.php

```diff
@@ -312,8 +310,6 @@ public function get_locale(): array
 			'SETTINGS_SUCCESS_IMAGE_OVERLAY' => 'EXIF-Overlay nastavenia aktualizované',
 			'SETTINGS_SUCCESS_PUBLIC_SEARCH' => 'Verejné vyhľadávanie bolo aktualizované',
 			'SETTINGS_SUCCESS_LICENSE' => 'Prednastavená licencia aktualizovaná',
-			'SETTINGS_SUCCESS_CSS' => 'CSS aktualizované',
-			'SETTINGS_SUCCESS_UPDATE' => 'Nastavenia úspešne aktualizované',
```

Comment:
Same thing here, see `settings.js` method `settings.changeCSS`. Don't delete it.